### PR TITLE
Improve claim method

### PR DESCRIPTION
**Pull request by svenvanderburg**
See resulting nanopublicaiton: http://test-server.nanopubs.lod.labs.vu.nl/RADSboX94R1mAPIZmAO6VxfNuLr50HYWRmzxonF6F5t9s

### CODE REVIEW

File: nanopub/client.py [hunk 1]
```diff
@@ -214,16 +214,35 @@ def publish(self, nanopub: Publication):
 
         return publication_info
 
-    def claim(self, text, rdftriple=None):
-        """
-        Publishes a claim, either as a plain text statement, or as an rdf triple (or both)
-        """
-        assertion_rdf = rdflib.Graph()
+    def claim(self, statement_text: str, author: str = None):
```
Comment (svenvanderburg) [inline nanopub/client.py:217]: I decided to only allow claiming with a statement_text and not with a triple even though that was mentioned in #19 . If you want to publish a triple you can use from_assertion method of Publication. This is just to quickly publish something.

File: nanopub/client.py [hunk 2]
```diff
@@ -214,16 +214,35 @@ def publish(self, nanopub: Publication):
...
+                                 'an ORCID iD (example: https://orcid.org/0000-0000-0000-0000)'
+                                 'for the author argument')
+            else:
+                author = rdflib.URIRef(profile.get_orcid_id())
+        else:
+            author = rdflib.URIRef(author)
+        publication = Publication.from_assertion(assertion_rdf=assertion_rdf,
+                                                 nanopub_author=author,
+                                                 attributed_to=author)
+        # TODO: This is a hacky solution, should be changed once we can add provenance triples to
```
Comment (svenvanderburg) [inline nanopub/client.py:244]: I added that this should be fixed in #36

**Comment by svenvanderburg** [comment]
Fixes #19

**Review by raar1 — APPROVED** [review]
All looks good to me. At some point in future maybe we will revisit the idea of AIDA sentences and do something here with them.

**Pull request merged**